pkg/gatewayserver/upstream/packetbroker/packetbroker.go

```diff
@@ -106,13 +106,20 @@ func (h *Handler) ConnectGateway(ctx context.Context, ids ttnpb.GatewayIdentifie
 	pbaClient := ttnpb.NewGsPbaClient(pbaConn)
 
 	gtw := conn.Gateway()
+	antennas := make([]*ttnpb.GatewayAntenna, len(gtw.Antennas))
+	for i, ant := range gtw.Antennas {
+		antennas[i] = &ant
+	}
```

Comment:
What is this for? Doesn't `antennas = append(antennas,&ant)` work? Items are appended in the order in which they are passed right?

Reply:
Yeah same amount of lines, same order; since we know the index this is slightly faster.